scripts/build_npm.ts

```diff
@@ -1,6 +1,23 @@
 import { build, emptyDir } from "https://deno.land/x/dnt/mod.ts";
+import { copy } from "https://deno.land/std@0.177.0/fs/mod.ts";
 
-await emptyDir("./npm");
+// Copy test data
+await Deno.remove("npm", { recursive: true }).catch((_) => {});
+await emptyDir("./npm/esm/src/m3/ecore/test");
+await emptyDir("./npm/script/src/m3/ecore/test");
+await emptyDir("./npm/esm/src/m3/test");
+await emptyDir("./npm/script/src/m3/test");
+
+await copy("diagrams", "npm/esm/diagrams", { overwrite: true });
+await copy("diagrams", "npm/script/diagrams", { overwrite: true });
+await copy("models", "npm/esm/models", { overwrite: true });
+await copy("models", "npm/script/models", { overwrite: true });
+await copy("schemas", "npm/esm/schemas", { overwrite: true });
+await copy("schemas", "npm/script/schemas", { overwrite: true });
```

Comment:
Does this mean that we have now 2 NPM packages: one for ESM and one for CommonJS?

Reply:
Yes. The CommonJS one is for Node and the ESM one is, I presume, for the browser (but I didn't test it in the browser and I would be surprised if it worked as-is). That's how [Deno's dnt tool](https://github.com/denoland/dnt) works.